internal/service/implement/auth_service.go

```diff
@@ -161,16 +161,18 @@ func (service *AuthService) SendOTPToMail(ctx *gin.Context) error {
 	otp := mail.GenerateOTP(6)
 
 	// store otp in redis for 1 minute
-	key := "otp"
-	err := service.redisCLient.Set(ctx, key, otp, 60)
+	customerId, _ := ctx.Get("customerId")
+	customerIdInt64, _ := customerId.(int64)
+	
+	baseKey := constants.REDIS_KEY
+	key := stringutils.Concat(baseKey, customerIdInt64)
```

Comment:
please create this utils in utils/redis. Since this is not normat concat, we need to add ":" as the delimeter for the key and the ID